mmseg/models/utils/post_process_layer.py

```diff
@@ -0,0 +1,66 @@
+import torch
+import torch.nn as nn
+
```

Comment:
Are the classes in this file necessary?

Reply:
using `torch.cat()` function and `nn.Module` class